app/src/main/res/values-night/string-not_translatable.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="pref_key__editor_basic_color_scheme__bg" translatable="false">@string/pref_key__editor_basic_color_scheme__bg_dark</string>
+    <string name="pref_key__editor_basic_color_scheme__fg" translatable="false">@string/pref_key__editor_basic_color_scheme__fg_dark</string>
+</resources>
```

Comment:
@gsantner We still have dark and light versions. It is just that the decision is made by the system, not in code